shop/src/main/java/ro/msg/learning/shop/exception/LocationNotFound.java

```diff
@@ -0,0 +1,7 @@
+package ro.msg.learning.shop.exception;
+
+public class LocationNotFound extends Exception{
```

Comment:
Exception is to general. This should be a RuntimeException. Same goes for the other custom Exception.